src/OrchardCore.Modules/OrchardCore.Contents/Views/Admin/List.cshtml

```diff
@@ -24,93 +29,75 @@
                     </div>
                 </div>
-                <div class="input-group-append">
-                    <span class="input-group-text bg-secondary text-light">@T["Select All"]</span>
-                </div>
+                <input asp-for="Options.DisplayText" class="form-control" placeholder="@T["Search all content items"]" type="search" />
             </div>
         </div>
-        <div class="form-group col-6 float-right">
-            <div class="dropdown float-right">
-                <button disabled class="btn btn-secondary dropdown-toggle" type="button" id="bulk-action-menu-button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                    @T["Bulk Actions"]
-                </button>
-                <div class="dropdown-menu dropdown-menu-right" aria-labelledby="bulk-action-menu-button">
-                    @foreach (var item in Model.Options.ContentsBulkAction)
-                    {
-                        <a class="dropdown-item" href="javascript:void(0)" data-action="@item.Value" data-title="@T["Bulk Action"]" data-message="@T["Are you sure you want to {0} this/these item(s)?", @item.Text.ToLower()]">@item.Text</a>
-                    }
-                </div>
+        <div class="form-group col-xl-8 col-sm-6">
+            <div class="btn-group float-right">
+                @if (Model.Options.CreatableTypes.Any())
+                {
+                    <button class="btn btn-sm btn-success dropdown-toggle" data-toggle="dropdown" id="new-dropdown" type="button" aria-haspopup="true" aria-expanded="false">New</button>
+                    <div class="dropdown-menu dropdown-menu-right" aria-labelledby="new-dropdown">
+                        @foreach (var item in Model.Options.CreatableTypes)
+                        {
+                            <a class="dropdown-item" href="ContentTypes/@item.Value/Create">@item.Text</a>
+                        }
+                    </div>
+                }
             </div>
         </div>
     </div>
     @if (Model.ContentItems.Count > 0)
     {
-        <ul class="list-group">
+        <ul class="list-group with-checkbox">
+            <li class="list-group-item bg-light">
+                <div class="row">
+                    <div class="form-group col-xl-3 mb-xl-n1">
+                        <div class="custom-control custom-checkbox mt-2 mr-n2">
+                            <input type="checkbox" class="custom-control-input" id="select-all">
+                            <label class="custom-control-label" for="select-all" id="select-all-label" title="@T["Select All"]"></label>
+                            <label id="items" for="select-all">@T.Plural(Model.ContentItems.Count, "1 item", "{0} items")<span class="small text-muted" title="@T["Items {0} to {1}", startIndex, endIndex]">@T.Plural((int)Model.Pager.TotalItemCount, " / {0} item in total", " / {0} items in total")</span></label>
```

Comment:
Again why we need to use the "small"?!!

Reply:
I used it when there was not enough space available. Now we have more space.